app/views/devise/passwords/edit.html.erb

```diff
@@ -18,8 +24,8 @@
   </div>
 
   <div class="actions">
-    <%= f.submit "Change my password" %>
+    <%= f.submit "Change my password", class: "btn btn-lg btn-primary btn-block" %>
```

Comment:
Just asking: are the app's consistently first-word-only-capitalized? If so, great.